### ISSUE TITLE
RNN Project: Misleading introduction to apple stock price normalization

### ISSUE THREAD

**Issue by ilona0925** [udacity/aind-issue-reports#196]
The introduction says "Here we normalize the series to lie in the range [0,1] using this scikit function" but the provided normalized data lie in the range [-1,1] and are obviously normalized with another function.

**Comment by letyrodridc** [comment]
In the notebook, "scikit function" is a link to  `MinMaxScaler` scikit function
The cell hasn't a TODO for the student to complete it and it doesn't call: `MinMaxScaler` function.
 
Besides that, I've found a code containing it:
```
%matplotlib inline
import numpy as np
import matplotlib.pyplot as plt
from sklearn.preprocessing import MinMaxScaler

### load in and normalize the dataset
dataset = np.loadtxt('datasets/apple_prices.csv')
scaler = MinMaxScaler(feature_range=(0, 1)) 
dataset = scaler.fit_transform(dataset.reshape(-1,1))
```

Was it intentionally removed and the comment left?

**Comment by ilona0925** [comment]
Also, in the first code cell of the notebook, these three lines are doubled, guess that's a mistake:
%load_ext autoreload
%autoreload 2

from my_answers import *

**Comment by ghost** [comment]
@letyrodridc I tried your code but it is still the same, do you have any other suggestion?

**Comment by letyrodridc** [comment]
Hi @haitrinh, 
About this waffle ticket, I've added the information for the person who needs to review the issue.
This issue is Open, not solved yet. It's about reviewing the comments in the notebook and the code.
Sorry, for the confusion. 
Besides that, the project can be completed anyway. 

Are you hai_1749979625607193 in forums? You have a different question: 
> I tried few but didn’t work, in the project https://github.com/udacity/aind2-rnn/blob/master/RNN_project.ipynb1 (1.1 Getting Started ) they left a link to MinMaxScaler but I can’t reproduce the same result as normalized_apple_prices.csv

Maybe @cgearhart  or an instructor can answer your question here.

**Comment by cgearhart** [comment]
Internal ticket #J1824
